src/sentry/replays/endpoints/project_replay_accessibility_issues.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+import uuid
+from typing import Any
+
+import requests
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.paginator import Cursor, CursorResult
+from sentry.models.project import Project
+from sentry.replays.lib.storage import make_filename
+from sentry.replays.usecases.reader import fetch_direct_storage_segments_meta
+
+REFERRER = "replays.query.query_replay_clicks_dataset"
+
+
+@region_silo_endpoint
+class ProjectReplayAccessibilityIssuesEndpoint(ProjectEndpoint):
```

Comment:
The build is failing because this class needs attribute owner. Checkout ApiOwner enum for values.